pkgs/development/python-modules/reproject/conftest.patch

```diff
@@ -0,0 +1,18 @@
+diff -ruN reproject-0.6.orig/reproject/conftest.py reproject-0.6/reproject/conftest.py
+--- reproject-0.6.orig/reproject/conftest.py	2019-11-01 16:57:57.000000000 +0100
++++ reproject-0.6/reproject/conftest.py	2020-02-25 17:39:07.000000000 +0100
+@@ -12,9 +12,11 @@
+ PYTEST_HEADER_MODULES['Astropy'] = 'astropy'
+ PYTEST_HEADER_MODULES['astropy-healpix'] = 'astropy_healpix'
+ PYTEST_HEADER_MODULES['Cython'] = 'cython'
+-del PYTEST_HEADER_MODULES['h5py']
+-del PYTEST_HEADER_MODULES['Matplotlib']
+-
++try:
++    del PYTEST_HEADER_MODULES['h5py']
++    del PYTEST_HEADER_MODULES['Matplotlib']
++except KeyError:
++    pass
```

Comment:
Looks similar to https://github.com/astropy/photutils/blob/master/photutils/conftest.py, so, perhaps, this can be submitted upstream?